SYCL/Regression/event_destruction.cpp

```diff
@@ -0,0 +1,35 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: %HOST_RUN_PLACEHOLDER %t.out
+// RUN: %CPU_RUN_PLACEHOLDER %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
```

Comment:
is there any reason to skip the execution of FPGA?

Reply:
I added run on FPGA